src/chainstate/stacks/index/storage.rs

```diff
@@ -185,16 +248,41 @@ impl TrieRAM {
         (lr, lw)
     }
 
+    pub fn write_trie_file<F: Write + Seek>(f: &mut F, node_data: &[(TrieNodeType, TrieHash)], offsets: &[u32],
```

Comment:
Should add a runtime `assert!` that `node_data` and `offsets` have the same length.

Reply:
Addressed in d31f0207